src/System.Data.Common/tests/System/Data/DataRelationCollectionTest2.cs

```diff
@@ -379,28 +379,18 @@ public void Add_DataColumn4()
             });
         }
 
+        //TODO: what
         [Fact]
         public void Add_DataColumn5()
         {
             DataSet ds = GetDataSet();
```

Comment:
what is this testing? looking for the private testing method to not throw? seems pointless.

Reply:
Is this method invoked elsewhere?  If yes, you can delete this test.  If no, you can bring the contents of it into this test method, assuming it's doing something meaningful.